src/main.rs

```diff
@@ -80,15 +82,17 @@ fn main() {
     let mut watcher: RecommendedWatcher = Watcher::new_raw(tx).unwrap();
     for m in monitor.as_vec().unwrap() {
         let path = m["path"].as_str().unwrap();
-        let ignore = match m["ignore"].as_str() {
-            Some(ig) => ig,
+        info!("Monitoring path: {}", path);
+        match m["ignore"].as_vec() {
+            Some(ig) => {
+                let ignore_list_vec  = ig.iter().map(|e| { e.as_str().unwrap() });
+                let ignore_list : String = Itertools::intersperse(ignore_list_vec, ", ").collect();
+                info!("Ignoring files with: {} inside {}", ignore_list, path);
```

Comment:
It is a good approach. We try to avoid new dependencies if we can. Try this approach and let me know if it match your requirements.
```suggestion
                info!("Ignoring files with: {:?} inside {}", ig, path);
```

Reply:
I mean I wanted to omit sth like this:
```
[INFO] Ignoring files with: [String(".log"), String(".txt")] inside /tmp/
```
but if it isn't problem to you, that's okay - we can push it.

Reply:
I think that itertools is good and supported dependency. We will include it.